README.md

```diff
@@ -18,17 +18,15 @@ If you use this to further your career, or put money in your pocket, and would l
 Documentation is located at http://benedmunds.com/ion_auth/
 
 ## Installation
-Just copy the files from this package to the corresponding folder in your
-application folder.  For example, copy Ion_auth/config/ion_auth.php to
-application/config/ion_auth.php
+Just copy the files from this package to the corresponding folder in your application folder. For example, copy Ion_auth/config/ion_auth.php to application/config/ion_auth.php
 
-You can also copy the libraries and models directories into your third_party/ion_auth folder.  For example, copy  to /application/third_party/ion_auth/.  The directory structure would be:
+You can also copy the libraries and models directories into your third_party/ion_auth folder.  For example, copy  to /application/third_party/ion_auth/. The directory structure would be:
 
     controllers/Auth.php
-    views/
-    third_party/ion_auth/libraries/Ion_auth.php
-    third_party/ion_auth/libraries/Bcrypt.php
-    third_party/ion_auth/models/Ion_auth_model.php
+    libraries/Ion_auth.php
```

Comment:
These are incorrect paths for third_party setup

Reply:
Maybe add two separate directory structure examples?

Reply:
I understand this repo is your baby. Simply trying to help to make things more clear. I am using this directory structure (if I followed the Readme there would be an additional **third_party** in the **third_party**) Hopefully, this screenshot will help you understand the change request further.

![Ion Auth Directory Structure](https://htmlgraphic.com/img/ion_auth_directory_structure.png)